import: improve file naming (#13038)
pingcap/docs: tidb-cloud/naming-conventions-for-data-import.md

When you import CSV files, name the data files as follows:

`${db_name}.${table_name}[${suffix}].csv`

`${suffix}` is optional and can be one of the following formats, where *`xxx`* can be any number:

- *`.xxx`*, such as `.01`

> - *`._xxx_xxx_xxx_`*, such as `._0_0_01`

Author: ```suggestion
- *`._xxx_xxx_xxx`*, such as `._0_0_01`
```

When you import CSV files, name the data files as follows:

> `${db_name}.${table_name}[${suffix}].csv`

Reviewer: ```suggestion
`${db_name}.${table_name}${suffix}.csv`
```

> `${db_name}.${table_name}[${suffix}].csv`

Reviewer 2: 需要在 ${suffix} 前面加个 . 吗？比如： 
```suggestion
`${db_name}.${table_name}[.${suffix}].csv`
```

Reviewer 3: 不需要，suffix 中也可以不包含 .，例如 _0_0_01

`${suffix}` is optional and can be one of the following formats, where *`xxx`* can be any number:

- *`.xxx`*, such as `.01`
- *`._xxx_xxx_xxx`*, such as `._0_0_01`
- *`_xxx_xxx_xxx`*, such as `_0_0_01`

> For example, all target databases and tables below are `import_db` and `test_table`:

Reviewer: ```suggestion
For example, the target database and table of all the following files are `import_db` and `test_table`:
```

- `import_db.test_table.csv`
- `import_db.test_table.01.csv`
- `import_db.test_table._0_0_01.csv`
- `import_db.test_table_0_0_01.csv`

### Parquet

When you import Parquet files, name the data files as follows:

> - `${db_name}.${table_name}[${suffix}].parquet[.{snappy|gz|lzo}]` (`${suffix}` and `[.{snappy|gz|lzo}]` are optional)

Reviewer: ```suggestion
- `${db_name}.${table_name}${suffix}.parquet.{snappy|gz|lzo}` (`${suffix}` and `.{snappy|gz|lzo}` are optional)
```

Author: ```suggestion
- `${db_name}.${table_name}${suffix}.parquet{.snappy|.gz|.lzo}` (`${suffix}` and `{.snappy|.gz|.lzo}` are optional)
```

When you import SQL files, name the data files as follows:

> - `${db_name}.${table_name}[${suffix}].sql` (`${suffix}` is optional)

Reviewer: ```suggestion
- `${db_name}.${table_name}${suffix}.sql` (`${suffix}` is optional)
```
